.eslintignore

```diff
@@ -1,24 +0,0 @@
-*.min.js
-/3rd-party/debug-bar/debug-bar.js
-/_inc/blocks
-/_inc/build
-/_inc/client/**/test/*.js
-/_inc/jetpack-modules.models.js
-/_inc/postmessage.js
-/docker/wordpress-develop/
-/docker/wordpress/
```

Comment:
We should keep these around somewhere, whether in a root-level .eslintignore or in tools/docker